.storybook/fetch-mocks.js

```diff
@@ -49,6 +50,18 @@ export function fetchMockSaveSettings() {
 	);
 }
 
+export function fetchMockSaveDataAvailable() {
```

Comment:
This is added to prevent warnings in stories about POSTing to the `data-available` endpoint once the gathering data state has been determined.